doc(release) Release notes for gorgone 24.10.1 (#3844)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-os.mdx

## Centreon Gorgone

### 24.10.1

Release date: `November 27, 2024`

<details>
  <summary>Bug fixes</summary>

> - [Gorgone] Avoid installation of centreon-engine and centreon-broker on MBI server.

Reviewer: ```suggestion
- [Gorgone] **centreon-engine** and **centreon-broker** are no longer installed on MBI servers.
```
